lib/content-services/src/lib/category/categories-management/categories-management.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="adf-categories-management">
+    <p *ngIf="!categories.length"
+       class="adf-no-categories-message">
+        {{  noCategoriesTitle | translate }}
+    </p>
+    <div class="adf-categories-list">
+        <span
+            *ngFor="let category of categories"
+            [class.adf-categories-padded]="!isCRUDMode"
+            class="adf-assigned-categories">
+            {{ category.name }}
+            <button
+                data-automation-id="categories-remove-category-button"
+                mat-icon-button
+                (click)="removeCategory(category)"
+                [attr.title]="removeCategoryTitle | translate"
+                [disabled]="disableRemoval">
+                <mat-icon>remove</mat-icon>
+            </button>
+        </span>
+    </div>
+    <div *ngIf="((!categoryNameControlVisible && categories.length)) ||categoryNameControlVisible"
+        [hidden]="!categoryNameControlVisible"
+        class="adf-category-name-field">
+        <mat-form-field>
+            <mat-icon matPrefix>search</mat-icon>
+            <mat-label id="adf-category-name-input-label">
+                {{'CATEGORIES_MANAGEMENT.NAME' | translate}}
+            </mat-label>
+            <input
+                matInput
+                autocomplete="off"
+                [formControl]="categoryNameControl"
+                (keyup.enter)="addCategory()"
+                aria-labelledby="adf-category-name-input-label"
+                adf-auto-focus
```

Comment:
Don't you need to import ContentDirectiveModule to use that directive?

Reply:
Right, imported